KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -18,7 +18,16 @@ default configuration = 'Debug'
 
 @{
     var projectFolder = Path.GetDirectoryName(projectFile);
```

Comment:
Yes. Somehow the rumor started that dotnet test didn't, but this was just a bug. See https://github.com/dotnet/coreclr.xunit/pull/7